modxcms/revolution
# [3.1] A11y/ManagerMenu: hotfix for links not clickable 3th level

### What does it do?

Hopefully it fixes #16667.

### How to test

1. Try clicking on all the links in the manager menu. Especially in the 3rd level.
2. Try navigating through the Manager Menu using the Tab key on your keyboard and clicking on the menu items using the Enter key.
3. Make sure there are no new errors 😸.  

### Related issue(s)/PR(s)

#16530
#16667
